cmd/derper/derper.go

```diff
@@ -166,7 +166,7 @@ func main() {
 		}
 		httpsrv.TLSConfig = certManager.TLSConfig()
 		go func() {
-			err := http.ListenAndServe(":80", certManager.HTTPHandler(tsweb.Port80Handler{mux}))
+			err := http.ListenAndServe(":80", certManager.HTTPHandler(tsweb.Port80Handler{Main: mux}))
```

Comment:
Normally I'd agree with this sort of change, but in this case this line we control both sides (so changes would break the build) and this is more mysterious here now: what other fields are there? Before it was obvious there was only one.

Reply:
Makes sense